_managing-content/collections.markdown

```diff
@@ -5,16 +5,13 @@ layout: page
 weight: 3
 ---
 
-There will be times where you don't need a page or a post, but still need to group bits of content together. This is exactly what _collections_ are: they are groups of [_documents_](/managing-content/documents/) that aren't in any particular order.
+There will be times where you don't need a page or a post, but still need to group bits of content together. This is exactly what collections are: they are groups of [documents](/managing-content/documents/) that aren't in any particular order.
 
-For each collection, a new directory will be created in your project called `_[collection name]/` (e.g., `_people/`). Additionally, a piece of configuration will be added to your `_config.yml` file that defines the collection with its title and some metadata:
+**Tip**: By default, a collection's documents aren't generated when you publish your site. If you'd like to output these documents, add a bit of [metadata](/managing-content/metadata) called `output` with the value `true`.
```

Comment:
🙅 We set `output` to `true` by default. Changing the collection's visibility to hidden will set it to `false`:

![image](https://cloud.githubusercontent.com/assets/371943/13606628/a3beb9b6-e51a-11e5-8957-73706751e0ab.png)